### ISSUE TITLE
Импорт данных с проходной.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2064]
Originally reported on Google Code with ID 2064

```
При импорте данных из дбф с проходной, на тех сотрудников, которые есть в дбф, но нет
в табеле отработанного времени (в гедымине) данные по таким не импортируются.  нужно
чтобы создавался на таких сотрудников автоматически табель и уже туда импортировались
данные из дбф.
```

Reported by `roberto1813` on 2010-06-28 08:14:09

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-07-01 05:29:42
- **Labels added**: Wage, VMK
- **Labels removed**: WageVMK

**Comment by gsbelarus** [comment]
```
Неактуально. Импорт с проходной теперь будет проходить во временную таблицу.
http://code.google.com/p/gedemin/issues/detail?id=2163
```

Reported by `alexandra.gsoftware` on 2010-10-14 18:55:35
- **Status changed**: `Duplicate`
- **Merged into**: #2163
